json-schema-autogen/assertion_evidence.json

```diff
@@ -85,6 +123,12 @@
             "title": "AssertionType",
             "type": "string"
         },
+        "AutomaticAssertion": {
+            "additionalProperties": false,
+            "description": "An assertion method that does not involve human review.",
```

Comment:
Generally speaking, but with some exceptions, I would like to avoid descriptions that classify via negation. Could this be: "An assertion method that is made via an automated process."? Perhaps if this is intended only to cover _in silico_ assertions, we could add a clause to that effect.

Reply:
I have updated the google sheet with _An assertion method executed automatically by machines without human intervention._ definition. So next time when we generate the model, it should replace this negated definition.